core/src/main/java/org/hibernate/ogm/dialect/impl/OgmDialect.java

```diff
@@ -31,6 +32,8 @@
 
 	public OgmDialect(GridDialect gridDialect) {
 		this.gridDialect = gridDialect;
+		//enabling support for binary types
+		registerColumnType( Types.BINARY, "binary" );
```

Comment:
This solution seems to work but I'm not convinced it's the right approach.
The problem seems to be that OgmDialect doesn't override `getDefaultMultiTableBulkIdStrategy()` and therfore it is using the one in the `Dialect` superclass. This is doing things that it shouldn't leading to the exception.

If I change the implementation, this test passes without the need to `registerColumnType`

Reply:
@DavideD What in the end will we do? I'll look at the solution through an override or you yourself?

Reply:
Can you have a look at `getDefaultMultiTableBulkIdStrategy`? I started with an empty implementation and it seems to pass the tests you added.

But we need some additional tests where we make sure that when we delete the entity the elements in all the tables are deleted.  Probably we have other scenarios to test, but we can start with this.

A simple way to do this is to create an entity, delete it and check that the db is empty.